packages/deployments/contracts/contracts/core/connext/helpers/RelayerProxy.sol

```diff
@@ -300,6 +342,40 @@ contract RelayerProxy is ProposedOwnable, ReentrancyGuard, GelatoRelayFeeCollect
     transferRelayerFee(_relayerFee);
   }
 
+  /**
+   * @notice Wraps the call to proposeAggregateRoot() on SpokeConnector and pays either the caller or hardcoded relayer
+   * @dev _rootTimestamp is required for off-chain agents to be able to know which root they should fetch from the root manager contract
+   *                     in order to compare it with the one being proposed. The off-chain agents should also ensure the proposed root is
+   *                     not an old one.
+   * @param _aggregateRoot The aggregate root to propose.
+   * @param _rootTimestamp Block.timestamp at which the root was finalized in the root manager contract.
+   * @param _signature Signature from the approved proposer.
+   * @param _fee - Fee to be paid to relayer.
+   */
+  function proposeAggregateRoot(
+    bytes32 _aggregateRoot,
+    uint256 _rootTimestamp,
+    bytes memory _signature,
+    uint256 _fee
+  ) external onlyRelayer nonReentrant {
+    if (!_proposeAggregateRootCooledDown()) {
+      revert RelayerProxy__proposeAggregateRootCooledDown_notCooledDown(
+        block.timestamp,
+        lastProposeAggregateRootAt + proposeAggregateRootCooldown
+      );
+    }
+
+    lastProposeAggregateRootAt = block.timestamp;
+
+    // Validate the signer
+    _validateProposeSignature(_aggregateRoot, _rootTimestamp, _signature);
+
+    spokeConnector.proposeAggregateRoot(_aggregateRoot, _rootTimestamp);
+
+    transferRelayerFee(_fee);
+    lastProposeAggregateRootAt = block.timestamp;
```

Comment:
duplicated assignment here and in line 368

Reply:
Done